### PR TITLE
Add some basic info to the README

### PR DESCRIPTION
**Pull request by inikulin**
\cc @AlexanderMoskovkin @MargaritaLoseva

### CODE REVIEW

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

File: README.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+#### Built-in reporters
+ - [testcafe-reporter-spec](https://github.com/DevExpress/testcafe-reporter-spec) - "Spec" reporter plugin.
+ - [testcafe-reporter-list](https://github.com/DevExpress/testcafe-reporter-list) - "List" reporter plugin.
+ - [testcafe-reporter-minimal](https://github.com/DevExpress/testcafe-reporter-minimal) - "Minimal" reporter plugin.
+ - [testcafe-reporter-json](https://github.com/DevExpress/testcafe-reporter-json) - "JSON" reporter plugin.
+ - [testcafe-reporter-xunit](https://github.com/DevExpress/testcafe-reporter-xunit) - "xUnit" reporter plugin.
+
+
+#### Plugin development tools
+- [generator-testcafe-reporter](https://github.com/DevExpress/generator-testcafe-reporter) - Yeoman generator for the reporter plugins.
```
Comment (MargaritaLoseva) [inline README.md]: Omit "the":
_Yeoman generator for reporter plugins._

**Comment by inikulin** [comment]
FPR

File: README.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+Tests             | Status
+----------------- | ----------
+All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+Client            | [![Sauce Test Status](https://saucelabs.com/browser-matrix/testcafe-master.svg)](https://saucelabs.com/u/testcafe-master)
+
+## Modules
+
+ - [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) - the heart.
+ - [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) - library for performing platform-dependent actions on browser windows.
```
Comment (MargaritaLoseva) [inline README.md]: _the_ library for performing platform-dependent actions on browser windows

File: README.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+ - [testcafe-legacy-api](https://github.com/DevExpress/testcafe-legacy-api) - legacy API support.
+ - [testcafe-gh-page-assets](https://github.com/DevExpress/testcafe-gh-page-assets) - website assets.
```
Comment (MargaritaLoseva) [inline README.md]: _the_ website assets

File: README.md [hunk 4]
```diff
@@ -1,5 +1,40 @@
...
+
+#### Built-in reporters
+ - [testcafe-reporter-spec](https://github.com/DevExpress/testcafe-reporter-spec) - "Spec" reporter plugin.
```
Comment (MargaritaLoseva) [inline README.md]: Before each reporter add _the_. For example:
_the_ "Spec" reporter plugin.

File: README.md [hunk 5]
```diff
@@ -1,5 +1,40 @@
...
+Tests             | Status
+----------------- | ----------
+All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+Client            | [![Sauce Test Status](https://saucelabs.com/browser-matrix/testcafe-master.svg)](https://saucelabs.com/u/testcafe-master)
+
+## Modules
+
+ - [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) - the heart.
+ - [testcafe-browser-natives](https://github.com/DevExpress/testcafe-browser-natives) - library for performing platform-dependent actions on browser windows.
+ - [testcafe-legacy-api](https://github.com/DevExpress/testcafe-legacy-api) - legacy API support.
```
Comment (MargaritaLoseva) [inline README.md]: _the_ legacy API support

**Comment by inikulin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: README.md [hunk 6]
```diff
@@ -1,5 +1,40 @@
...
+#### Built-in reporters
+ - [testcafe-reporter-spec](https://github.com/DevExpress/testcafe-reporter-spec) - the "Spec" reporter plugin.
+ - [testcafe-reporter-list](https://github.com/DevExpress/testcafe-reporter-list) - the "List" reporter plugin.
+ - [testcafe-reporter-minimal](https://github.com/DevExpress/testcafe-reporter-minimal) - the "Minimal" reporter plugin.
+ - [testcafe-reporter-json](https://github.com/DevExpress/testcafe-reporter-json) - the "JSON" reporter plugin.
+ - [testcafe-reporter-xunit](https://github.com/DevExpress/testcafe-reporter-xunit) - the "xUnit" reporter plugin.
+
+
+#### Plugin development tools
+- [generator-testcafe-reporter](https://github.com/DevExpress/generator-testcafe-reporter) - Yeoman generator for reporter plugins.
```
Comment (MargaritaLoseva) [inline README.md]: Please, add "the" before Yeoman:
_The_ Yeoman generator for reporter plugins

**Comment by inikulin** [comment]
FPR

**Comment by MargaritaLoseva** [comment]
lgtm

**Pull request merged**